src/main/java/figure/Square.java

```diff
@@ -0,0 +1,57 @@
+package figure;
+
+public class Square extends AbstractFigure {
+    private double side;
+
+    public Square(double side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    public Square(int side, String color) {
+        this.side = side;
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure is square{" +
+                " side =" + side +
+                ", area =" + getArea() +
+                ", color='" + color + '\'' +
+                ", unique method is getDiagonal = " + getDiagonal() +
+                '}';
```

Comment:
```suggestion
        return "Figure is square{" +
                " side=" + side +
                ", area=" + getArea() +
                ", color='" + color + '\'' +
                ", unique method is getDiagonal= " + getDiagonal() +
                '}';
```
make your formatting uniform